### ISSUE TITLE
System check: trans.IndexUpdate.unit

### ISSUE THREAD

**Issue by nblock** [nijel/weblate#733]
Hi,
I just noticed the following:
```
/var/www/weblate-venv/bin/python /var/www/weblate/manage.py check
System check identified some issues:

WARNINGS:
trans.IndexUpdate.unit: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.

System check identified 1 issue (0 silenced).
```
Python 2.7.9
Weblate 2.2

**Comment by nijel** [comment]
I know this happens in Django 1.8. I haven't yet find time if we can safely migrate the field or if it will cause some problems...

**Comment by nijel** [comment]
Fixed in c5a2bbb9
